eq-publisher/src/eq_schema/builders/routing2/translateBinaryExpression.test.js

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable camelcase */
+
+const translateBinaryExpression = require("./translateBinaryExpression");
+
+describe("Should build a runner representation of a binary expression", () => {
+  it("should throw on unsupported answer type", () => {
+    const expression = {
+      left: {
+        id: "1",
+        type: "Text"
+      },
+      condition: "Equal",
+      right: {
+        number: 5,
+        __typeName: "CustomValue2"
+      }
+    };
+    try {
```

Comment:
It would be cleaner to use one of the following method `Jest` provides:

> // const fn = () => { throw new Error('Out of cheese!') }
> expect(fn).toThrow()
> expect(fn).toThrow('Out of cheese')
> expect(fn).toThrowErrorMatchingSnapshot()

https://github.com/sapegin/jest-cheat-sheet/blob/master/Readme.md